extensions/v2ex-viewer/src/utils/markdown.ts

```diff
@@ -0,0 +1,42 @@
+import { Topic, Reply } from "../api/types";
+import { getUnixFromNow } from "./time";
+
+const SEPARATOR = `\n\n------\n\n`;
+const LINE_BREAK = `\n\n`;
+
+const Code = (content: string) => `\`${content}\``;
+const Bold = (content: string) => `**${content}**`;
+const Link = (title: string, url: string) => `[${title}](${url})`;
+const Image = (alt: string, url: string) => `![${alt}](${url})`;
+const Heading = (level: number, content: string) => {
+  level = Math.max(1, Math.min(6, level));
+  return `${"#".repeat(level)} ${content}`;
+};
+
+const OP = (isOP: boolean) => {
+  return isOP ? Code("OP") : "";
+};
+
+const getTopicMarkdownContent = (topic: Topic, replies: Reply[], includeHeader = false) => {
+  const topicTitle = `${Heading(1, topic.title)}`;
+  const topicMember = `${Bold(Link(topic.member.username, topic.member.url))} · ${getUnixFromNow(topic.created)} · ${
+    topic.replies
+  } 条回复`;
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.